WordPress/src/main/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModel.kt

```diff
@@ -182,15 +188,29 @@ class ActivityLogViewModel @Inject constructor(
         }
     }
 
+    // todo: annmarie - Remove once the feature exclusively uses the more menu
     fun onActionButtonClicked(item: ActivityLogListItem) {
         if (item is Event) {
-            if (backupFeatureConfig.isEnabled()) {
-                // todo: annmarie implement expand/dismiss the more menu
-                _showMoreMenu.value = item
-            } else {
-                _showRewindDialog.value = item
+            _showRewindDialog.value = item
+        }
+    }
+
+    fun onSecondaryActionClicked(
+        secondaryAction: ActivityLogListItem.SecondaryAction,
+        item: ActivityLogListItem
+    ): Boolean {
+        if (item is Event) {
+            val navigationEvent = when (secondaryAction) {
+                RESTORE -> {
```

Comment:
Minor (🔍 ): This and the below lines can become one-liners. Consider removing braces from the `when` entries.